engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/inputSettings/InputSettingsScreen.java

```diff
@@ -27,11 +27,8 @@
 import org.terasology.engine.module.ModuleManager;
 import org.terasology.engine.subsystem.config.BindsManager;
 import org.terasology.i18n.TranslationSystem;
-import org.terasology.input.BindButtonEvent;
-import org.terasology.input.Input;
-import org.terasology.input.InputCategory;
-import org.terasology.input.InputSystem;
-import org.terasology.input.RegisterBindButton;
+import org.terasology.input.*;
```

Comment:
Terasology style strongly avoids star imports. Please change this back to importing items individually. [If you are using IntelliJ then this guide will help](https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports)

Reply:
Thank you, I must have missed this guideline.
This has now been corrected.